### PR TITLE
{bio} [goolf-1.4.10] Stacks 1.35 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 2
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/e23aaa369c61a9bc3f0e for a full test report.

File: easybuild/easyconfigs/s/Stacks/Stacks-1.19-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+#
+# Full Stacks pipeline is php + mysql + command line utilies
+# This will install JUST the command line utilities
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/Stacks/Stacks-1.19-goolf-1.4.10.eb]: the above has been added; please recheck zlib need and test/merge as appropriate

**Comment by fgeorgatos** [comment]
put this in REVIEW plz!

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7433dc6952094ea548b2 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
missing dep: `google-sparsehash/2.0.2-goolf-1.4.10`

**Comment by boegel** [comment]
Jenkins: test this please (is the dep there now?)

**Comment by boegel** [comment]
(close/open to trigger Travis)

File: easybuild/easyconfigs/s/Stacks/Stacks-1.35-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ population genomics and phylogeography. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://creskolab.uoregon.edu/stacks/source/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('SAMtools', '0.1.19'),
+    ('google-sparsehash', '2.0.2'),
```
Comment (boegel) [inline easybuild/easyconfigs/s/Stacks/Stacks-1.35-goolf-1.4.10.eb]: @pescobar name was changed to `sparsehash` a long time ago, see #1452, please fix?

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**